lua/entities/gmod_wire_expression2/core/wirelink.lua

```diff
@@ -107,6 +107,17 @@ local function WriteArray(entity, address, data)
 	return free_address
 end
 
+local function writeArraySimple(entity, address, data)
+	local written = 0
+	for index, value in pairs(data) do
+		if type(value) == "number" then
```

Comment:
Yes I know it's not relevant *here*.

Anyway, sorry for not getting back to this earlier, yes it's ok to merge.